package.json

```diff
@@ -209,6 +209,7 @@
     "@babel/preset-typescript": "7.13.0",
     "@babel/register": "7.13.8",
     "@graphql-inspector/core": "1.14.0",
+    "@graphql-tools/jest-transform": "^1.2.0",
```

Comment:
this plugin looks better than the other, let's see for how long they'll maintain it 😂

Reply:
yah that's a good question. I had to upgrade because of a jest breaking change that the other lib hasn't switched over to, but so far so good.